### DISCUSSION TITLE
选中文字后右键菜单全选按钮缺少图标

### DISCUSSION THREAD

**Discussion by TPAXcc** [ywmoyue/biliuwp-lite#593] [General]
### Bug 描述

![image](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/1d0279a5-ec56-4eb2-a4e0-3fde6f1dcba8)

### 操作步骤

```plain text
双击选中后右键

另外还有个BUG 
手动选中文字之后右键较大概率弹不出来着个框
```

### 期望结果

![image](https://github.com/ywmoyue/biliuwp-lite/assets/91980905/4bd6f6cd-1f90-4a15-9a39-411dcfe6c947)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

Windows 11 21H2 (Build 22000), 其它 (需注明)

### 其它系统版本

版本	Windows 11 专业工作站版 版本号	24H2 安装日期	‎2024/‎2/‎10 操作系统版本	26052.1000 体验	Windows Feature Experience Pack 1000.26052.1000.0

### 设备类型

台式机

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
这个问题不是应用本身的问题而是WinUI的问题，这个右键菜单不是应用中实现的，将系统语言设置为英文后这个右键菜单和WinUI2Gallery的右键菜单一模一样

![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/858f8f12-4936-411c-b0ec-722c1b74fd33)

**Comment by ywmoyue** [comment]
所以不做处理，转discussion
